app/views/provider/admin/account/payment_gateways/braintree_blue/edit.html.slim

```diff
@@ -94,11 +94,12 @@ p
   = form.actions do
     = form.commit_button 'Save credit card'
 
+/ TODO: All is duplicated. Move this into a pack and use braintree from node_modules
```

Comment:
If memory serves right, this single bar is a comment in `Slim`. Still weird to me 😬

Reply:
Comment, correct